### ISSUE TITLE
Doc is not up to date

### ISSUE THREAD

**Issue by pierrbt** [SequoiaSan/FlipperZero-WiFi-Scanner_Module#17]
https://github.com/SequoiaSan/FlipperZero-WiFi-Scanner_Module#how-to-add-new-application-on-flipperzero

I tried to add the application to Firmware but [applications/meta/application.fam](https://github.com/flipperdevices/flipperzero-firmware/blob/dev/applications/meta/application.fam) do not exists anymore.

**Comment by SequoiaSan** [comment]
Hi pierrbt!

Thank you for pointing this out. I haven't had enough time last days, because of the war here in Ukraine, but I'll post a new release of Scanner as a standalone app and update the Readme.

**Comment by pierrbt** [comment]
You mean a .fap file ?
And ok i understand, good Luck for the War 🇺🇦

**Comment by SequoiaSan** [comment]
Yes, .fap file. Haven't had a chance to look how now they propose to compile apps in *.fap
When this happen I'll post new release + update docs accordingly.

Thanks! ❤️
